extensions/devservices/oidc/src/main/java/io/quarkus/devservices/oidc/OidcDevServicesProcessor.java

```diff
@@ -593,19 +593,19 @@ private static void refreshTokenEndpoint(RoutingContext rc) {
         String clientSecret = rc.request().formAttributes().get("client_secret");
         String scope = rc.request().formAttributes().get("scope");
         if (clientId == null || clientId.isEmpty()) {
-            LOG.warn("Invalid client ID, denying token refresh");
+            LOG.warn("client_id is not present, denying token request");
             invalidTokenResponse(rc);
             return;
         }
         if (clientSecret == null || clientSecret.isEmpty()) {
-            LOG.warn("Invalid client secret, denying token refresh");
+            LOG.warn("client_secret is not present, denying token request");
             invalidTokenResponse(rc);
             return;
         }
         String refreshToken = rc.request().formAttributes().get("refresh_token");
         UserAndRoles userAndRoles = decode(refreshToken);
         if (userAndRoles == null) {
-            LOG.warn("Received invalid refresh token, denying token refresh");
+            LOG.warnf("Received invalid refresh_token, denying token refresh: %s", refreshToken);
```

Comment:
`refresh_token` is a JSON property name, I think using `refresh token` is OK here

Reply:
Similarly for `client_id` and `client_secret` - it implies users know that on the wire the client id and secret are represented these property names. IMHO, it is worth keeping it less technical `Client id is not present` ,`Client secret is not present`

Reply:
ok, I am at +0 about this topic, if the technical name or a nicer user friendly name should be used.
Might be nicer to have the field name in the log message, if one is already familiar with OIDC. 

But maybe lets wait for more feedback; I guess I will just change it back :)